src/components/FilterPanel.js

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core";
+import { useTheme } from '@material-ui/core/styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Switch from "@material-ui/core/Switch";
+import Select from "@material-ui/core/Select";
+import MenuItem from "@material-ui/core/MenuItem";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+
+
+const useStyles = makeStyles((theme) => ({
+    formControlLabel: {
+        "text-align": "left",
+        "align-items": "start",
+        "margin-top": theme.spacing(1),
+        "width": `calc(100% - ${theme.spacing(4)}px)`
+    },
+    mdPanel: {
+        "position": "sticky",
+        "top": 0,
+        "height": "200px"
+    }
+}));
+
+export default function FilterPanel(props) {
+    const classes = useStyles();
+    const theme = useTheme();
+    const mdSize = useMediaQuery(theme.breakpoints.up('md'));
+
+    const [hasAppointments, setHasAppointments] = useState(true);
+    const [vaxType, setVaxType] = useState('Any');
+
+    const { data, onChange } = props;
+
+    useEffect(() => {
+        onChange({
+            hasAppointments: d => !hasAppointments || !!d.hasAppointments,
+            vaxType: d => {
+                if (vaxType === 'Any') {
+                    return true;
+                } else {
+                    if (d.extraData && d.extraData['Vaccinations offered']) {
+                        return d.extraData['Vaccinations offered'].includes(vaxType);
+                    } else {
+                        return false;
+                    }
+                }
+            }
+        })
+    }, [onChange, hasAppointments, vaxType])
+
+    const vaxTypes = Array.from(
```

Comment:
I'm a little nervous here. Only the MA vaccination sites will have that "Vaccinations offered" information that we scrape up; no other sites will have it. Therefore by default we'll show them in the "Any" bucket which is misleading. It might be worth entering manually what sites (outside of the state sites) have which vaccines, but then that's something we would have to manually update if it ever changes.

Curious if you (or anyone else) have thoughts.